circuits/cpp/src/aztec3/utils/types/circuit_types.hpp

```diff
@@ -50,6 +51,10 @@ template <typename Composer> struct CircuitTypes {
     // typedef bn254::g1_ct bn254_point;
     typedef stdlib::element<Composer, fq, fr, barretenberg::g1> bn254_point; // affine
 
+    typedef stdlib::secp256k1<Composer> secp256k1;
+    typedef typename secp256k1::bigfr_ct secp256k1_fr;
+    typedef typename secp256k1::g1_bigfr_ct secp256k1_point;
```

Comment:
g1_bigfr_ct is a confusing name - I haven't looked at what this means:
Is it a group E(F_q), whose points are (Fq, Fq), which has group size Fr? (Where Fr is secp256k1_fr)
Or is it the group E(Fr), whose points are (Fr, Fr)?